src/components/admin/ScraperDashboard.tsx

```
@@ -0,0 +1,230 @@
+               className="w-full p-2 border rounded dark:bg-gray-700"
+             />
+          </div>
+        </div>
+
+        <button 
+          onClick={handleStart} 
+          disabled={loading || activeStatus?.status === 'running'}
+          className={`px-4 py-2 rounded text-white font-medium ${
+            activeStatus?.status === 'running' ? 'bg-gray-400 cursor-not-allowed' : 'bg-blue-600 hover:bg-blue-700'
+          }`}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)